runtime/client_http_request.go

```diff
@@ -128,6 +125,9 @@ func (req *ClientHTTPRequest) Do(
 		return nil, err
 	}
 
+	// emit metrics
+	req.metrics.Sent.Inc(1)
```

Comment:
Did you want to move this to `finish()` instead?  Just keep it with all the other emitted metrics as I had it earlier.

Reply:
this should be right after we write arg3 to the request.

Reply:
Or for this HTTP case; it should be right here at 128 after we send an outbound http request.